### PR TITLE
Fixed relocation of projects

### LINKED ISSUE #4772: Controller not found after relocation
**Describe the Bug**

In the latest nightly (tar), relocating the `highway_driving` sample prints an additional warning concerning the controller. Since this warning was not present in this issue: https://github.com/cyberbotics/webots/issues/4726, I suspect the issue was recently added.

```
ERROR: JavaScript error: could not open file: /home/daniel/Downloads/highway_driving/worlds/forest/highway_overtake/1.forest.
WARNING: DEF WEBOTS_VEHICLE0 LincolnMKZ (PROTO): The 'automobile' robot window library has not been found.
WARNING: DEF WEBOTS_VEHICLE0 LincolnMKZ (PROTO): The controller directory has not been found, searched the following locations:
/home/daniel/Downloads/highway_driving/controllers/highway_driving/
/home/daniel/Downloads/webots-R2022b-x86-64/webots/projects/vehicles/protos/lincoln/controllers/highway_driving/
/home/daniel/Downloads/webots-R2022b-x86-64/webots/projects/default/controllers/highway_driving/
/home/daniel/Downloads/webots-R2022b-x86-64/webots/resources/projects/controllers/highway_driving/
```

### PR DESCRIPTION
**Pull request by omichel**
Fixes  #4772.

This PR fixes the missing copies of controller files as well as SUMO files.

### CODE REVIEW

File: src/webots/editor/WbProjectRelocationDialog.cpp [hunk 1]
```diff
@@ -224,67 +224,29 @@ void WbProjectRelocationDialog::copy() {
...
-  if (copyProtoProject) {
-    // copy all PROTO textures
-    result += WbFileUtil::copyDir(projectPath + "protos/textures", mTargetPath + "/protos/textures", true, true, true);
-    result += WbFileUtil::copyDir(projectPath + "protos/meshes", mTargetPath + "/protos/meshes", true, true, true);
-    result += WbFileUtil::copyDir(projectPath + "protos/skins", mTargetPath + "/protos/skins", true, true, true);
-  }
-
-  bool projectLibrariesCopied = false;
+  // copy all local resources
+  result += WbFileUtil::copyDir(projectPath + "protos", mTargetPath + "/protos", true, false, true);
```
Comment (ad-daniel) [inline src/webots/editor/WbProjectRelocationDialog.cpp:231]: More of a question really, does the project relocation action trigger only for our own folders? If so can't we skip copying the protos folder? It should be empty no? (well, technically it shouldn't be there to begin with after the cleanup of the distribution)

**Review by ad-daniel — CHANGES_REQUESTED** [review]
Works well, just a question

File: src/webots/editor/WbProjectRelocationDialog.cpp (hunk 1 above)
Comment (omichel) [reply src/webots/editor/WbProjectRelocationDialog.cpp:231]: Yes, good point. I can remove this copy of this `protos` folder indeed.

Comment (omichel) [reply src/webots/editor/WbProjectRelocationDialog.cpp:231]: Finally, I would prefer to keep it like that for the following reasons:
1. At some point we may include a protos folder in our example to demonstrate how to have a local PROTO.
2. Some people may fork Webots and add their own projects which may include a protos folder.
3. Some people may distribute some project folder which installs without write rights, and thus users would need to relocate it, including possibly a protos folder.

**Review by ad-daniel — APPROVED** [review]
Thank you

**Pull request merged**